lib/archethic/contracts/interpreter/library.ex

```diff
@@ -9,6 +9,66 @@ defmodule Archethic.Contracts.Interpreter.Library do
   alias Archethic.P2P.Message.GenesisAddress
   alias Archethic.P2P.Message.FirstPublicKey
 
+  @doc """
+  Returns the element at given index (zero-based)
+
+  ## Examples
+
+    iex> Library.at([1,2], 1)
+    2
+
+    iex> Library.at([], 0)
+    nil
+
+    iex> Library.at([], 0, "hello")
+    "hello"
+  """
+  def at(list, idx, default \\ nil) do
+    Enum.at(list, idx, default)
+  end
+
+  @doc """
+  Convert a string to an integer
+
+  ## Examples
+
+    iex> Library.int("12345")
+    12345
+  """
+  @spec int(String.t()) :: integer()
+  def int(text) do
```

Comment:
I think the function name could be more explicit so that anybody could understand by reading the code. Maybe something like `string_to_int`, `string_to_float`, `number_to_string`.